Base/usr/share/man/man7/Help-index.md

```diff
@@ -2,20 +2,23 @@
 
 ## Welcome to the SerenityOS on-line help system!
 
-This is **Help**, the built-in documentation viewer for the SerenityOS desktop environment. If you prefer a command-line interface, the [`man`(1)](help://man/1/man) command offers a text-only view of the same library.
+This is the system documentation for SerenityOS, in the form of [manual pages](https://en.wikipedia.org/wiki/Man_page) divided into traditional Unix-style [manual categories](https://en.wikipedia.org/wiki/Man_page#Manual_sections). If you want more information on the structure of the manual system, take a look at [`man`(7)](help://man/7/man).
 
----
+Use the [Help](help://man/1/Help) application to access the man pages within SerenityOS. If you prefer a command-line interface, the [`man`(1)](help://man/1/man) command offers a text-only view of the same library. It is also available as a website at [man.serenityos.org](https://man.serenityos.org/).
 
-Documentation is divided into traditional Unix-style manual categories. Use the **Browse** tab on the left-hand side to navigate by category.
+In the Help application, use the **Browse** tab on the left-hand side to navigate by category. There's also a full-text search option under the **Search** tab.
 
-There's also a full-text search option under the **Search** tab.
+---
 
-If you want more information on the structure of the manual system, take a look at [`man(7)`](help://man/7/man).
+- [Section 1: User Programs (applets, applications, utilities, etc.)](man1/index.html)
+- [Section 2: System Calls (getuid, mount, pledge, sendfd, etc.)](man2/index.html)
+- [Section 3: Library Functions (basename, isatty, POSIX functions, etc.)](man3/index.html)
+- [Section 4: Special Files (audio, mem, etc.)](man4/index.html)
+- [Section 5: File Formats (getopt convention, Shell, SystemServer)](man5/index.html)
+- [Section 6: Games (2048, Chess, FlappyBug, etc.)](man6/index.html)
+- [Section 7: Miscellanea (mitigations, sys filesystem, SystemServer, etc.)](man7/index.html)
+- [Section 8: Sysadmin Tools (dmesg, pls, sysctl, useradd, etc.)](man8/index.html)
```

Comment:
We should have this information auto-generated, but there's no better way than this at the moment.

Reply:
I'm stuck on how to fix the links for this - how do we list them as `help://` URLs that get converted?

Reply:
The other problem is to get pandoc to treat the index like the other pages so `help://` URLS are transformed and the right paths are set for images. I'm afraid this is technically above me (although I'm trying my best)